go/bundle/cmd/gen-bundle/main.go

```diff
@@ -20,7 +20,7 @@ import (
 var (
 	flagHar      = flag.String("har", "", "HTTP Archive (HAR) input file")
 	flagDir      = flag.String("dir", "", "Input directory")
-	flagBaseUrl  = flag.String("baseUrl", "https://example.com/", "Base URL")
+	flagBaseUrl  = flag.String("baseUrl", "", "Base URL")
```

Comment:
`flagBaseURL` for Go convention https://stackoverflow.com/questions/38616687/which-way-to-name-a-function-in-go-camelcase-or-semi-camelcase

Reply:
done